### PR TITLE
Ui improvements

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
MM-11121 - Updating searchbar icon and add team icon
MM-11122 - Updating styling of emoji picker 
MM-11111 - Fixing expand/collapse icon for posts
MM-11112 - Updating post textarea icons

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-11111
https://mattermost.atlassian.net/browse/MM-11121
https://mattermost.atlassian.net/browse/MM-11122
https://mattermost.atlassian.net/browse/MM-11112

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Ran `make check-style` to check for style errors (required for all pull requests)
- [ ] Ran `make test` to ensure unit and component tests passed
- [ ] Added or updated unit tests (required for all new features)
- [ ] Has server changes (please link)
- [ ] Has redux changes (please link)
- [x] Has UI changes
- [ ] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates
- [ ] Touches critical sections of the codebase (auth, posting, etc.)

### CODE REVIEW

File: components/emoji_picker/emoji_picker.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ import EmojiPickerPreview from './components/emoji_picker_preview';
 
 const CATEGORY_SEARCH_RESULTS = 'searchResults';
 const EMOJI_HEIGHT = 27;
-const EMOJI_CONTAINER_HEIGHT = 244;
+const EMOJI_CONTAINER_HEIGHT = 290;
```
Comment (sudheerDev) [inline components/emoji_picker/emoji_picker.jsx]: This add's some empty space between emoji picker button and gyfcat container which is not the case on pre-release.  

screen shot 2018-07-03 at 11 28 53 am

Comment (asaadmahmood) [reply components/emoji_picker/emoji_picker.jsx]: Yup, I noticed that.  Is there a way we can increase the width of the images a bit?

Comment (asaadmahmood) [reply components/emoji_picker/emoji_picker.jsx]: Should be fixed now @sudheerDev. Have a look.
screen shot 2018-07-03 at 5 14 29 pm

Comment (sudheerDev) [reply components/emoji_picker/emoji_picker.jsx]: @asaadmahmood i meant the space from emoji picker button to the emoji picker has increased.
screen shot 2018-07-03 at 6 05 55 pm

this height is hard coded here at https://github.com/mattermost/mattermost-webapp/blob/91a518aa363cf913bf1f544f3bbf56f787aadcce/components/gif_picker/components/SearchGrid/SearchGrid.scss#L7

For reference: changing that to `336px` did the trick for me.

Comment (asaadmahmood) [reply components/emoji_picker/emoji_picker.jsx]: Oh I see. Fixed @sudheerDev

**Pull request merged**